rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1435,6 +1435,31 @@ protected void doJUMPDEST()
         program.step();
     }
 
+    protected void doMCOPY() {
+        if (computeGas) {
+            // See "Gas Cost" section on EIP 5656
+            // gas cost = 3 * (length + 31) + memory expansion cost + very low
+            long length = stack.get(stack.size() - 3).longValue();
+            long newMemSize = memNeeded(stack.peek(), length);
+            long cost = 3 * (length + 31) + calcMemGas(oldMemSize, newMemSize, 0) + 3; // TODO -> Check copy size
```

Comment:
In order to calculate the number of words we are missing to divide the total size by the dataword length being `(length + 31) / 32` instead of only `(length + 31) `.
Also I think it is more readable if we store the different values separated adding a meaningful name to them

Reply:
Nice! will continue with this in the next PR. 💪🏼